packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -1034,3 +1055,101 @@ function getPeriodStartTime(
+      const winningReportingParticipantId = await market.getWinningReportingParticipant_();
+      const winningReportingParticipant = augur.contracts.getReportingParticipant(winningReportingParticipantId);
+      if (!stakeDetails[payoutDistributionHash]) {
+        // Create new StakeDetails for Payout Set
+        const payout = await reportingParticipant.getPayoutNumerators_();
+        let bondSizeCurrent: BigNumber;
+        let stakeCompleted: BigNumber;
+        if (stakeLogs[i].hasOwnProperty("amountStaked")) {
+          bondSizeCurrent = new BigNumber(stakeLogs[i].amountStaked);
+          stakeCompleted = bondSizeCurrent;
+        } else {
+          bondSizeCurrent = new BigNumber(stakeLogs[i].size);
+          stakeCompleted = disputeCrowdsourcerCompletedLogs[0] ? new BigNumber(disputeCrowdsourcerCompletedLogs[0].size) : new BigNumber(0);
+        }
+        stakeDetails[payoutDistributionHash] =
+          {
+            payout,
+            isInvalid: payout.length > 0 && payout[0].gt(0) ? true : false,
+            bondSizeCurrent,
+            bondSizeTotal: bondSizeCurrent,
+            stakeCurrent,
+            stakeRemaining,
+            stakeCompleted,
+            tentativeWinning: await winningReportingParticipant.getPayoutDistributionHash_() === payoutDistributionHash ? true : false,
+          };
+      } else {
+        // Update existing StakeDetails for Payout Set
+        if (disputeCrowdsourcerCompletedLogs[0]) {
+          stakeDetails[payoutDistributionHash].stakeCompleted = new BigNumber(stakeDetails[payoutDistributionHash].stakeCompleted).plus(disputeCrowdsourcerCompletedLogs[0].size);
+        } else {
+          stakeDetails[payoutDistributionHash].bondSizeCurrent = new BigNumber(stakeLogs[i].size);
+          stakeDetails[payoutDistributionHash].stakeCurrent = stakeCurrent;
+        }
+        if (new BigNumber(stakeLogs[i].size).gt(stakeDetails[payoutDistributionHash].bondSizeTotal)) {
+          stakeDetails[payoutDistributionHash].bondSizeTotal = stakeDetails[payoutDistributionHash].bondSizeTotal.plus(stakeLogs[i].size);
+        }
+        if (stakeRemaining.gt(0)) {
+          stakeDetails[payoutDistributionHash].stakeRemaining = new BigNumber(stakeRemaining);
+        }
+      }
+    }
+  }
+
+  return {
+    disputePacingOn: await market.getDisputePacingOn_(),
+    stakeCompletedTotal: (await market.getParticipantStake_()).toString(10),
+    bondSizeOfNewStake: (await market.getParticipantStake_()).times(2).toString(10),
+    stakes: formatStakeDetails(Object.values(stakeDetails)),
+  };
+}
+
+function formatStakeDetails(stakeDetails: any[]): StakeDetails[] {
+  const formattedStakeDetails: StakeDetails[] = [];
+  for (let i = 0; i < stakeDetails.length; i++) {
+    formattedStakeDetails[i] = {
+      payout: convertPayoutNumeratorsToStrings(stakeDetails[i].payout),
```

Comment:
can the getter also add outcome along with payout. I don't think the UI needs to work with payout numerators

Reply:
Sure, I'll add that in now.